### PR TITLE
Install the latest version of the CLI

### PR DESCRIPTION
**Pull request by banool**
## Summary
This way we don't need to keep updating the versions and releasing this package.

## Test Plan
Tested on MacOS and Linux. It should work on Windows too since the behavior with Linux and Windows is identical.

### CODE REVIEW

File: bin/aptos [hunk 1]
```diff
@@ -37,16 +36,29 @@ const executableIsAvailable = (name) => {
   }
 }
 
+const getLatestVersionGh = async () => {
+  const response = await (await fetch("https://api.github.com/repos/aptos-labs/aptos-core/releases?per_page=100")).json();
```
[the remark this answers is not in the document]
Comment (banool) [reply bin/aptos:40]: I think so, `fetch` just comes from node so it should be fine.

File: bin/aptos [hunk 2]
```diff
@@ -60,25 +72,35 @@ const main = async () => {
...
+    let latestVersion;
+    if (os === "MacOS") {
+      latestVersion = getLatestVersionBrew();
+    } else {
+      latestVersion = await getLatestVersionGh();
+    }
+
+    console.log(`Downloading aptos CLI version ${latestVersion}`);
+
+    const url = `https://github.com/aptos-labs/aptos-core/releases/download/${PNAME}-v${latestVersion}/${PNAME}-${latestVersion}-${os}-x86_64.zip`;
```
Comment (0xmaayan) [inline bin/aptos:90]: url is used only on windows/linux right?

**Review by 0xmaayan — COMMENTED** [review]
lgtm. I think we might want to include some comments explaining what are we doing - especially OS considerations

File: bin/aptos (hunk 2 above)
Comment (banool) [reply bin/aptos:88]: Yep, I could change the conditional logic so we have a "if windows or linx branch". But just making this const here is nbd.

**Comment by banool** [comment]
Just added some more comments.

**Review by 0xjinn — APPROVED** [review]

**Pull request merged**